client/src/main/kotlin/io/provenance/client/grpc/AbstractPbClient.kt

```diff
@@ -168,13 +183,13 @@ open class AbstractPbClient<T : ManagedChannelBuilder<T>>(
     fun estimateAndBroadcastTx(
         txBody: TxOuterClass.TxBody,
         signers: List<BaseReqSigner>,
-        mode: ServiceOuterClass.BroadcastMode = ServiceOuterClass.BroadcastMode.BROADCAST_MODE_SYNC,
+        mode: BroadcastMode = BroadcastMode.BROADCAST_MODE_SYNC,
         gasAdjustment: Double? = null,
         feeGranter: String? = null,
         feePayer: String? = null,
         txHashHandler: PreBroadcastTxHashHandler? = null,
         signatures: List<ByteArray?> = List(signers.size) { null },
-    ): ServiceOuterClass.BroadcastTxResponse =
+    ): ListenableFuture<BroadcastTxResponse> =
```

Comment:
I made these listenable as that seems handy for the caller 🤷

Reply:
In the non-coroutine `PbClient` you may not want to deal with futures at all and may want to just make this blocking.

Reply:
Yeah I was talking about this with Phil and think you're right, I'll go ahead and change this back to blocking so it just keeps the same signature as before. Shouldn't really make a difference from the caller's perspective

Reply:
pushed that up